frontend/src/components/user-profile/profile.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from "react";
+import "./profile.css";
+
+class ProfilePage extends React.Component {
+  render() {
+    return (
+      <div className="card">
+        <div className="card-body">
+          <div className="container emp-profile">
+            <form method="post">
+              <div className="row">
+                <div className="col-md-4">
+                  <div className="profile-img">
+                    <img
+                      src="https://scontent-dus1-1.xx.fbcdn.net/v/t31.0-8/21273280_1869007506448034_8133580273270929214_o.jpg?_nc_cat=105&_nc_ht=scontent-dus1-1.xx&oh=b8cacb7402c7d2745280b584f48f30c7&oe=5D5DAADA"
+                      alt=""
+                    />
+                    <div className="file btn btn-lg btn-primary">
```

Comment:
maybe we can make the change photo visible when the mouse is hover over the photo.